.gitignore

```diff
@@ -162,7 +162,8 @@ browsers-report/
 lit-browsers-report/
 visual-report/
 lit-visual-report/
-react-browsers-report
+react-browsers-report/
+packages/components/**/playwright/
```

Comment:
do you need to do the cache remove thing we spoke about last week?

Reply:
Nope, this should be fine :)